packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/numberBoxParts/mask.tests.js

```diff
@@ -2087,13 +2087,15 @@ QUnit.module('format: caret boundaries', moduleConfig, () => {
 
         this.input.focus();
         this.clock.tick(CARET_TIMEOUT_DURATION);
-        for(let i = 0; i < 2; ++i) {
-            this.keyboard.caret(3);
-            this.input.trigger('dxclick');
-            this.clock.tick(CARET_TIMEOUT_DURATION);
-        }
 
+        this.keyboard.caret(3);
```

Comment:
Tbh i don't know why loop was used here but I also don't see a reason to change it.
Did u check that this test didn't become evergreen after your change?

Reply:
it seems, this test checks returning caret after click. So after my fix,two dxclick fire dxdblclick therefore loop is not correct test here

Reply:
And still, did u check that this test is not evergreen now? It was added during a bug fix

Reply:
i add back second click in this tests, it seems tests needs two clicks